buildpack/databroker/business_events.py

```diff
@@ -30,16 +34,16 @@ def update_config(m2ee, vcap_services_data):
     logging.debug("Business Events config added to MicroflowConstants")
 
 
-def _get_client_config(url, auth_token, version):
+def _put_client_config(url, auth_token, dependencies_json):
     headers = {
-        "Authorization": f"Bearer {auth_token}",
-        "X-Version": version,
```

Comment:
any specific reason you feel versioning is not required now?

Reply:
yeah good question, i guess it was me being weak not to know on which version to start so i removed it all together. Version 1 is now for the get, should this be a version 1 PUT? or version 2 PUT? what do you think?